app/src/main/res/values-land/dimens.xml

```diff
@@ -395,4 +395,9 @@
   
   <!-- StoryFragment -->
   <dimen name="story_fragment_padding_bottom">60dp</dimen>
+
+  <!-- TopicLessonsTitle -->
+  <dimen name="topic_lessons_title_margin_start">100dp</dimen>
+  <dimen name="topic_lessons_title_margin_top">32dp</dimen>
+  <dimen name="topic_lessons_title_margin_end">100dp</dimen>
```

Comment:
`topic_lessons_title_margin_top` is same for both portrait and landscape so you can remove it from here

Reply:
okk

Reply:
As a reviewer we need to see our comments to verify that the requested changes are done or not. So remeber to not resolve them on your own.